ranger/container/directory.py

```diff
@@ -252,6 +252,19 @@ def hidden_filter_func(fobj):
                         return False
                 return True
             filters.append(hidden_filter_func)
+        if self.settings.global_inode_type_filter and not self.inode_type_filter:
+            # pylint: disable=too-many-boolean-expressions
+            def hidden_inode_filter_func(obj):
+                g_filter = self.settings.global_inode_type_filter
+                if (
+                        ('d' in g_filter and obj.is_directory) or
```

Comment:
That sounds like a good idea. I can easily refactor both the local and global inode_filters into one block of code; which would actually take care of resolving any conflicts between the two.

I'll work on that code in case. But I too would like to hear @hut's opinion.